src/styles/page.scss

```diff
@@ -0,0 +1,36 @@
+.page {
+  display: flex;
+  flex-direction: column;
+
+  &__header {
+    display: flex;
+    padding: 0 50px;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  }
+  
+  &__logo {
+    height: 40px;
+  }
+
+  &__main {
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+    padding: 50px 40px;
+  }
+
+  &__row {
+    display: flex;
+    
+    &:not(:last-child) {
+      margin-bottom: 46px;
+    }
+  }
```

Comment:
let's remove separate rows, what if there will be more than 8 cards? then you would need to create a new row instead of adding just another card to container

Reply:
Changed layout to a one container